src/routes.ts

```diff
@@ -3,10 +3,21 @@ import userControllers = require('./controllers/users');
 
 class routes{
 
-    constructor(app:Restify.Server){
+    constructor(server:Restify.Server){
+
+        server.post("/users/login/", userControllers.login);
+        server.get("/users/logout", userControllers.logout);
+        
+        /*********** Documentation routes ***********/
         
-        app.post("/users/login/", userControllers.login);
-        app.get("/users/logout", userControllers.logout);
+        // /docs does not render the css correctly, so redirect to /docs/
+        server.get('/docs', function(req, res, next){
+            res.header('Location', '/docs/');
+            res.send(302);
+        });
+
+        server.get(/\/docs\/?.*/, Restify.serveStatic({directory: './swagger',default: 'index.html'}));
```

Comment:
Oh cool, I knew you could redirect that way in express but I didn't see anything in the restify docs.  Also, can you elaborate on what you mean by add them to config.ts?  You mean just a variable that houses their file path?